components/main/GoalCard/GoalCard.tsx

```
@@ -0,0 +1,119 @@
+        subGoals={goalWithSubGoalTodo?.subGoals.map((subGoalInfo) => ({
+          id: subGoalInfo.subGoalId ?? "",
+          title: subGoalInfo.subGoal ?? "",
+        }))}
+        // modal이 등장하면 bottomSheet는 닫기.
+        openBottomSheet={
+          !isModalOpened && goalWithSubGoalTodo?.subGoals.length > 0
+        }
+        onSubmitTodo={async (newTodoInfo) => {
+          const isCreating = newTodoInfo.id ? false : true;
+          let fetchRes;
+          if (isCreating) {
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)